cdap-kubernetes/src/main/java/io/cdap/cdap/master/environment/k8s/KubeMasterEnvironment.java

```diff
@@ -150,8 +150,8 @@
   private static final String RESOURCE_QUOTA_NAME = "cdap-resource-quota";
   private static final String WORKLOAD_IDENTITY_ENABLED = "master.environment.k8s.workload.identity.enabled";
   private static final String WORKLOAD_IDENTITY_POOL = "master.environment.k8s.workload.identity.pool";
-  private static final String WORKLOAD_IDENTITY_GCP_SERVICE_ACCOUNT_EMAIL
-    = "master.environment.k8s.workload.identity.gcp.service.account.email";
+  @VisibleForTesting
+  static final String WORKLOAD_IDENTITY_GCP_SERVICE_ACCOUNT_EMAIL_PROPERTY = "gcp.service.account.email";
```

Comment:
This is GCP specific change in k8s area of code, anyway to avoid ?

Reply:
Or can we just name this service.account.email ? 

Understand that workload identity kind of already suggests this is GCP specific

Reply:
Yup, this is GCP specific, already had a discussion with chtyim@ regarding #13797 and he mentioned it should be fine to use a flag-gated feature since it does not have any actual Google Java client library dependencies.